core/src/AzureMcp.Cli/Program.cs

```diff
@@ -63,6 +63,7 @@ private static IAreaSetup[] RegisterAreas()
             new AzureMcp.Core.Areas.Tools.ToolsSetup(),
 
             // Register Azure service areas
+            new AzureMcp.Acr.AcrSetup(),
```

Comment:
@jongio, could you please pull from main and run ` ./eng/scripts/Build-Local.ps1 -BuildNative` to verify whether the new Acr project is AOT safe. If any errors occur (very likely), pls follow these steps:

1.	Move the `AcrSetup()` under the `#if !BUILD_NATIVE` block in Program.cs.
2.	Add ProjectReference-Remove in `AzureMcp.Cli.proj` when the `BuildNative` condition is true, (e.g., similar to Cosmos).

CI will be configured to execute this script in BuildNative mode next week so we can auto detect.

Reply:
That needs to be added to #new-command.md  I'm in there so I will make the update.

Reply:
I got this updated, but I think I'm missing something because the test is failing that it can't find the tool.  Can you have a look?